core/src/main/java/com/godaddy/commerce/core/http/HttpClientCache.java

```diff
@@ -110,6 +111,9 @@ private static CloseableHttpClient createClient(CacheKey key) {
       builder.setDefaultCredentialsProvider(provider);
     }
 
+    if(key.httpClientBuilderCustomizer != null)
+      key.httpClientBuilderCustomizer.accept(builder);
```

Comment:
> [<img alt="mtbennett" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.secureserver.net/mtbennett) **Authored by [mtbennett](https://github.secureserver.net/mtbennett)**
_<time datetime="2022-10-28T15:51:53Z" title="Friday, October 28th 2022, 5:51:53 pm +02:00">Oct 28, 2022</time>_

---


```suggestion
    if (key.httpClientBuilderCustomizer != null) {
      key.httpClientBuilderCustomizer.accept(builder);
    }
```

Reply:
> [<img alt="lhostetler" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.secureserver.net/lhostetler) **Authored by [lhostetler](https://github.secureserver.net/lhostetler)**
_<time datetime="2022-10-28T18:36:26Z" title="Friday, October 28th 2022, 8:36:26 pm +02:00">Oct 28, 2022</time>_

---


Agreed. Other than the quick "return" or maybe "continue", I like single statements in a code block too.